### PR TITLE
Portfolio2

### PR DESCRIPTION
**Pull request by Tracy-miranja**
I just created my final mobile app portfolio.check it out

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -33,19 +33,134 @@ <h2>I’am a Software Developer</h2>
...
+                <img src="./images/Img Placeholder.png" alt="project1">
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="list">
+                    <li class="list1"><a href="#">css</a></li>
+                    <li class="list1"><a href="#">html</a></li>
+                    <li class="list1"><a href="#">bootstrap</a></li>
+                    <li class="list1"><a href="#">Ruby</a></li>
+                </ul>
+                <div class="project"><a href="#">see project</a></div>
```
Comment (Yazino12) [inline index.html:50]: - [ ] Again you did well adding the hover effects to this button ✔, but the `font-type` and `color` you are using here is not consistent with the overall design 🤔. Please fix this 👇🏽

### Figma design:
image
### Your design:
image

File: index.html [hunk 2]
```diff
@@ -33,19 +33,134 @@ <h2>I’am a Software Developer</h2>
...
+                </div>
+            </div>
+        </div>
+        <div class="About">
+            <div class="aboutme">
+                <h1>About me</h1>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <div class="resume">Get my Resume</div>
+
+
```
Comment (Yazino12) [inline index.html:131]: - [ ] Please always try to pay attention to the design details 🥏, sticking to the details is what makes your whole portfolio stick out. As you see in the screenshots below you need to: `space your content in the center` `use the correct font for the text` and `use the correct color for the Get my Resume button` 👇🏽

### Figma design:
image
### Your design:
image

**Review by Yazino12 — CHANGES_REQUESTED** [review]
## Hi @Tracy-miranja 🖐🏾,

You've done well implementing most of the project requirements 👏🏽, but there are still some changes you need to make before merging it 🚀🚀🚀.

### To highlight:
- Descriptive PR title and summary ✔️
- Well organized code ✔️

# STATUS: CHANGES REQUIRED ♻♻♻
CHANGE
_Check the comments under the review 👇🏽👇🏽👇🏽._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @Tracy-miranja , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

chelsea-mason-approve



## Highlights
All linters are passing :heavy_check_mark: 
Followed correct GitHub flow
Well organised code :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**